rules/user/lookup.yara

```diff
@@ -1,14 +0,0 @@
-rule getpwuid : harmless {
```

Comment:
Were these rules supposed to be added to `rules/user/current.yara`?

Reply:
good catch that these were accidentally removed - re-added them.